lib/CGI.pm

```diff
@@ -931,7 +931,7 @@ sub _setup_symbols {
 
 	if ( /^[:-]any$/ ) {
 		warn "CGI -any pragma has been REMOVED. You should audit your code for any use "
-			. "of none supported / incorrectly spelled tags and remove them"
+			. "of unsupported/incorrectly-spelled tags and remove them"
```

Comment:
needless churn, please remove

Reply:
This is not needless: "none supported" is incorrect English, and it's not obvious what it means. But as I said, I'm happy to put it in another pull request, so I'll do that.

Reply:
In the context of the error message there is no ambiguity in what "none supported" means here.

Reply:
I've moved this fix to PR #223, and updated this PR.